app/questionnaire/questionnaire_store_updater.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         router: Router,
         current_question: Mapping | None,
     ):
+        self.evaluated_dependents: list[tuple] = []
```

Comment:
This shouldn't be an attribute of QSU. It is only used as temporary storage within `update_progress_for_dependent_sections` and it's inner calls.

Reply:
Reverted this change 👍